apps/meteor/client/views/room/body/RoomBodyV2.tsx

```diff
@@ -81,7 +81,7 @@ const RoomBody = (): ReactElement => {
 		return subscribed;
 	}, [allowAnonymousRead, canPreviewChannelRoom, room, subscribed]);
 
-	const innerBoxRef = useRef<HTMLDivElement | null>(null);
+	const innerBoxRef = useRef<HTMLElement>(undefined);
```

Comment:
Is `innerBoxRef` still needed here? It doesn't seem to be in use besides being merged at L121.